correios/models/posting.py

```diff
@@ -13,22 +13,29 @@
 # limitations under the License.
 
 
-import os
 import math
+import os
```

Comment:
I don't like alphabetical ordering. I prefer to import in the same sequence as we use modules/packages on code.

Reply:
@osantana isort has this [settings](https://github.com/timothycrosley/isort/wiki/isort-Settings). How do you want it?